### ISSUE TITLE
WCAG 2.5.8: Target Size (Minimum)

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5156]
**Level:** AA
**Principle:** Operable
**Result:** *pending*

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/XXXXX.html)

### Audit Checklist
- [ ] Review previous Audit Result
- [ ] Review work identified in previous audit and list open issues.
- [ ] Check for any other relevant open issues.
- [ ] Retest open issues, are these issues still present?
- [ ] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
 - link previous audit and summarise what was find and the outcome.

 ### Progress since last audit

 ### Audit Notes

 ### Work arising

**Comment by StephDriver** [comment]
## Test results

Generated: 2026-03-19T16:29:31.341Z
URLs tested: 65
Total focusable elements: 5298

These tables shows the failures only.  

### Clean

| Page                | a  | button | input | ul | Total Failures|
| --------------------------- | -- | ------ | ----- | -- | ----------- |
| olh/accessibility/          |    | 1      |       |    | 1           |
| olh/search/                 |    |        | 6     |    | 6           |
| olh/collections/            |    | 3      |       |    | 3           |
| olh/news/429/               |    | 1      |       |    | 1           |
| olh/articles/               |    |        | 56    |    | 56          |
| olh/article/id/4405/        |    | 6      |       |    | 6           |

### Material 
| Page                | a  | button | input | ul | Total Failures|
| --------------------------- | -- | ------ | ----- | -- | ----------- |
| olh/article/id/4405/        | 1  |        |       |    | 1           |

### OLH
| Page                | a  | button | input | ul | Total Failures|
| --------------------------- | -- | ------ | ----- | -- | ----------- |
| press-home                  | 3  |        |       | 1  | 4           |
| 404/                        | 2  |        |       |    | 2           |
| journals/                   | 38 |        |       |    | 38          |
| contact/                    | 2  |        |       |    | 2           |
| olh/                        | 6  |        |       |    | 6           |
| olh/accessibility/          | 1  |        |       |    | 1           |
| olh/contact/                | 1  | 1      |       |    | 2           |
| olh/editorialteam/          | 1  |        |       |    | 1           |
| olh/search/                 | 1  |        | 6     |    | 7           |
| olh/site/about/             | 1  |        |       |    | 1           |
| olh/site/author-guidelines/ | 1  |        |       |    | 1           |
| olh/site/journal-policies/  | 1  |        |       |    | 1           |
| olh/submissions/            | 1  |        |       |    | 1           |
| olh/issues/                 | 1  |        |       |    | 1           |
| olh/issue/402/info/         | 1  |        |       |    | 1           |
| olh/issue/409/info/         | 1  |        |       |    | 1           |
| olh/collections/            | 1  | 3      |       |    | 4           |
| olh/collections/846/        | 1  |        |       |    | 1           |
| olh/news/                   | 13 |        |       |    | 13          |
| olh/news/429/               | 1  | 1      |       |    | 2           |
| olh/articles/               | 1  | 5      | 56    |    | 62          |
| olh/article/id/4403/        | 1  |        |       |    | 1           |
| olh/article/id/4405/        | 2  | 14     |       |    | 16          |
